ui/index.html

```diff
@@ -106,6 +106,7 @@
         <script src="js/models/option_collection.js"></script>
         <script src="js/models/user.js"></script>
         <script src="js/models/error.js"></script>
+        <script src="js/perf.js"></script>
         <!-- Controllers -->
```

Comment:
Yes, if you look at perf.js, we're adding PhSeries to the perf module (https://github.com/mozilla/treeherder/blob/master/ui/js/perf.js#L5).

It's probably time to split out PhSeries and move it to its own file in ui/js/models/ and add it to the treeherder module instead. As long as we import it in perf.html we should be fine, since the perf module depends on the treeherder module.

You might need to do a few other things as part of this process but I think that covers most of it. :)